bigquery/cloud-client/natality_tutorial.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def run_natality_tutorial():
+    # [START bigquery_query_natality_tutorial]
+    """Create a Google BigQuery linear regression input table.
+
+    In the code below, the following actions are taken:
+    * A new dataset is created "natality_regression."
+    * A new table "regression_input" is created to hold the inputs for our
+        linear regression.
+    * A query is run against the public dataset,
+        bigquery-public-data.samples.natality, selecting only the data of
+        interest to the regression, the output of which is stored in the
+        "regression_input" table.
+    * The output table is moved over the wire to the user's default project via
+        the built-in BigQuery Connector for Spark that bridges BigQuery and
+        Cloud Dataproc.
+    """
+
+    from google.cloud import bigquery
+
+    # Create a new Google BigQuery client using Google Cloud Platform project
+    # defaults.
+    client = bigquery.Client()
+
+    # The name for the new dataset.
+    dataset_id = 'natality_regression'
+
+    # Prepare a reference to the new dataset.
+    dataset_ref = client.dataset(dataset_id)
+    dataset = bigquery.Dataset(dataset_ref)
+
+    # Create the new BigQuery dataset.
+    dataset = client.create_dataset(dataset)
+
+    # In the new BigQuery dataset, create a new table.
+    table_ref = dataset.table('regression_input')
+    # The table needs a schema before it can be created and accept data.
```

Comment:
I actually think the query job will run just fine without a schema, since BigQuery knows all they correct types when it runs the query.